src/Service/CognitoIdentityProvider/tests/Unit/Result/ResendConfirmationCodeResponseTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace AsyncAws\CognitoIdentityProvider\Tests\Unit\Result;
+
+use AsyncAws\CognitoIdentityProvider\Result\ResendConfirmationCodeResponse;
+use AsyncAws\CognitoIdentityProvider\ValueObject\CodeDeliveryDetailsType;
+use AsyncAws\Core\Response;
+use AsyncAws\Core\Test\Http\SimpleMockedResponse;
+use AsyncAws\Core\Test\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpClient\MockHttpClient;
+
+class ResendConfirmationCodeResponseTest extends TestCase
+{
+    public function testResendConfirmationCodeResponse(): void
+    {
+        // see https://docs.aws.amazon.com/cognitoidentityprovider/latest/APIReference/API_ResendConfirmationCode.html
+        $response = new SimpleMockedResponse(\json_encode([
+            'CodeDeliveryDetails' => [
+                'AttributeName' => 'a',
+                'DeliveryMedium' => 'b',
```

Comment:
Coul you please use Enum `DeliveryMediumType`?

Reply:
fixed. By the way, I have noticed that there is a `changelog` file, should I add info about new endpoint to it or is it updated automatically?

Reply:
I'll apreciate it (see how other operation are handled).

Thanks

Reply:
Other PR's are already merged.. Is it ok if I put all the new endpoints to this PR as a separate commit?

Reply:
anyway, pushed as a separate commit. If you think that it has to be as a single commit - say and I will do change